OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -18,6 +19,8 @@ namespace OpenRA.Mods.Common.Activities
 {
 	public class ReturnToBase : Activity
 	{
+		const int TooCloseDestinationThreshold = 5000;
```

Comment:
The required value is going to depend on the aircraft speed.  It should be calculated from that instead of being specified anywhere.